.github/actions/build-pandas/action.yml

```diff
@@ -5,14 +5,23 @@ runs:
   steps:
 
     - name: Environment Detail
+      id: env-detail
       run: |
+        echo ::group::{Environment Detail}
         conda info
         conda list
+        echo ::set-output name=python-version::$(conda list -f python --json | jq -r '.[0].version')
```

Comment:
The conda environment is activate by the time this runs correct?

Reply:
Yep!